config.local.yaml

```diff
@@ -4,6 +4,6 @@ chain_id: "testing"
 rpc_endpoint: "http://localhost:26657/"
 grpc_endpoint: "http://localhost:9090/"
 wsrpc_endpoint: "ws://localhost:26657/websocket"
-contract_address: "juno14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9skjuwg8"
+contract_address: null # This can be set via CRONCAT_CONTRACT_ADDRESS, committing this is a pain.
```

Comment:
Nice comment

Reply:
@SeedyROM i agree its a pain for local, but we need to make sure we dont lose the official addresses for each network. Agents shouldn't be responsible for config, we will need to publish versions of the official addresses.

So actually, (probably diff issue), is there a better way to manage each chains contract addresses? 
We will have to maintain at least 1 address per supported network.

Reply:
@TrevorJTClarke Right, I agree. I'm allowing for the override manually. If the config has the contract addr and no env var is set it uses the config address.

Reply:
Also local environments will always have different contract addresses right? How could you deploy a contract on a new chain at the same addr?

Reply:
I'm not sure about another way to manage the chain addresses outside of config. I'm open to suggestions though!